scripts/common.sh

```diff
@@ -165,7 +166,9 @@ function node_cmd {
   echo -n " --kademlia-dump-path  $(dump_path $kademlia_dump_path)"
   echo -n " --system-start $system_start"
   echo -n " --peer-id $peer_id"
-  # echo -n " --monitor-port $monitor_port +RTS -T -RTS "
+  echo -n " --metrics +RTS -T -RTS"
```

Comment:
We can if we want build in the +RTS -T param at compile time. I assume that would be preferable.